bin/crew

```diff
@@ -1616,7 +1617,7 @@ def resolve_dependencies_and_build
   @resolve_dependencies_and_build = 0
 end
 
-def build_package(pwd)
+def build_package(crew_archive_dest)
```

Comment:
Why not capitalized to reference the constant?

Reply:
good point. changed.

Reply:
Turns out that doesn't work as it fails the linter check:
```ruby
bin/crew:1620:19: F: Lint/Syntax: formal argument cannot be a constant
(Using Ruby 3.1 parser; configure using TargetRubyVersion parameter, under AllCops)
def build_package(CREW_ARCHIVE_DEST)
```